### DISCUSSION TITLE
Skipping some time periods on a time based graph - DUPLICATE

### DISCUSSION THREAD

**Discussion by zgh-public** [GSI-CS-CO/chart-fx#461] [Q&A] [answered]
Hallo. Great thanks for the nice product.

I'm drawing some local business time based data, and there is no data during holidays and nights. So I'd like to skip these large empty spaces on the graph. I've found de.gsi.chart.axes.spi.DefaultNumericAxis#axisTransform. I believe it is a feature I need. But I see no way to set axisTransform into the DefaultNumericAxis. There is only a get method.
How can I skip some time periods?

Best regards, Andrey

**Comment by RalphSteinhagen** [comment, marked as answer]
Hi @zgh-public there have been some previous [discussions](https://github.com/GSI-CS-CO/chart-fx/issues/294#issuecomment-721765150-permalink) on this while discussing financial charts. Unfortunately, there isn't a default implementation for this since this is use-case and specific to a given locale. However, we are happy to incorporate this if you are willing to provide one via a PR.

I am locking this here since there is already an open discussion and hints regarding this [here](https://github.com/GSI-CS-CO/chart-fx/discussions/394). If you could comment there if you have some further ideas/suggestions.
